### PR TITLE
examples: Added Greedy algorithm in Python.

### PR DESCRIPTION
**Pull request by JakubNvk**
Signed-off-by: Jakub Novak <jakubn@imterra.com>
Reviewed-by: Peter Polacik <pepol@imterra.com>

### CODE REVIEW

File: examples/example.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    amounts.append(int(input()))
+
+  for amount in amounts:
+    result_one = greedy(amount)
+    for i in range(1, 5):
+      if (amount - 4700*i) >= 0:
+        result_two = i + greedy(amount - 4700*i)
+        if result_one > result_two:
+          result_one = result_two
+
```
Comment (pepol) [inline examples/example.py]: Is this blank line needed? Because I think the indentation alone does enough...

Comment (JakubNvk) [inline examples/example.py]: You're right.

Comment (pepol) [inline examples/example.py]: No, I'm Peter :-D (so sorry)

**Comment by pepol** [comment]
LGTM, thanks @JakubNvk

**Pull request merged**